modules/telco-ran-workload-partitioning.adoc

```diff
@@ -1,26 +1,27 @@
 // Module included in the following assemblies:
 //
-// * scalability_and_performance/telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+// * scalability_and_performance/telco_ran_du_ref_design_specs/telco-ran-du-rds.adoc
 
 :_mod-docs-content-type: REFERENCE
 [id="telco-ran-workload-partitioning_{context}"]
-= Workload partitioning
+= Workload Partitioning
```

Comment:
Wondering why init cap? Esp when later usage in the body text is staying lowercase.

Reply:
Typo :)